nncf/common/pruning/pruning_node_selector.py

```diff
@@ -241,25 +241,25 @@ def _is_module_prunable(self, graph: NNCFGraph, node: NNCFNode) -> Tuple[bool, s
         types_to_track = self._prune_operations + stop_propagation_ops
         input_non_pruned_nodes = get_first_nodes_of_type(graph, types_to_track)
         output_non_pruned_nodes = get_last_nodes_of_type(graph, types_to_track)
-        module_identifier = get_module_identifier(node)
+        node_name = node.node_name
 
-        if not should_consider_scope(module_identifier, self._target_scopes, self._ignored_scopes):
-            msg = 'Ignored adding Weight Pruner in: {}'.format(module_identifier)
+        if not should_consider_scope(node_name, self._target_scopes, self._ignored_scopes):
```

Comment:
True, the matching is no longer being done on a prefix basis since we currently do not guarantee that our node names are prefix-free in a sense. Still, I think that the config names of `ignored_scopes` and `target_scopes` could be left as-is, since they do define certain *scopes* in a literary sense which should be ignored or targeted, and these scopes are defined in terms of node names.

Reply:
But we still use `Scope` class in PT. How to distinguish where node_name and where PT scope should be used?

Reply:
This is easy - the `should_consider_scope` fn will accept a limited number of serializable (str-able) objects specified by type hints, please review once I push.